lmcache/cache_engine.py

```diff
@@ -231,6 +238,7 @@ def store(
         self,
         tokens: torch.Tensor,
         kv_tensors_raw: KVCache,
+        kv_tensors_mask: Optional[torch.Tensor] = None,
```

Comment:
What's the assumption you made for kv_tensors_mask? I can see 2 different designs here:
1. `kv_tensors_mask`'s length is the same as number of tokens in in the `kv_tensors_raw`
2. number of `True`s in `kv_tensors_mask` is the same as number of tokens in the `kv_tensors_raw`

Can you also make it explicit in the docstring?

Reply:
And will the `kv_tensors_mask` have arbitrary True/False in it? Or the Falses/Trues can only be the prefix?